packages/web/src/components/userprofile/UserProfile.tsx

```diff
@@ -29,7 +29,7 @@ export const UserProfile: React.FC<UserProfileProps> = ({ user, setUser }: UserP
         'content-type': 'application/json',
       },
       body: JSON.stringify({
-        discordId: undefined,
+        discordId: null,
```

Comment:
Why was this change made? `null` doesn't match the type above (`string` in conjunction with `?` becomes `string | undefined` which is not the same as `string | null`)

Reply:
This was made because I don't think the database knows what undefined was. Unless we change it to null the discord ID is not actually removed from the profile. it just selects the profile but doesn't actually remove the Discord ID.